vocs.config.ts

```diff
@@ -0,0 +1,71 @@
+import { defineConfig } from "vocs";
+
+export default defineConfig({
+  title: "Open Frames",
+  sidebar: [
+    {
+      text: "Specification",
+      link: "/",
+    },
+    {
+      text: "Quickstart",
+      collapsed: false,
+      items: [
+        {
+          text: "frames.js",
+          link: "/quickstart/frames-js",
+        },
+        {
+          text: "OnchainKit",
+          link: "/quickstart/onchainkit",
+        },
+      ],
+    },
+    {
+      text: "Best Practices",
+      link: "/best-practices",
+    },
+    {
+      text: "Ecosystem",
+      collapsed: false,
+      items: [
+        {
+          text: "XMTP",
+          link: "/ecosystem/xmtp",
+        },
+        {
+          text: "Lens",
+          link: "/ecosystem/lens",
+        },
+        {
+          text: "Farcaster",
+          link: "/ecosystem/farcaster",
+        },
+      ],
+    },
+    {
+      text: "Examples",
+      collapsed: false,
+      items: [
+        {
+          text: "Awesome Open Frames ↗︎",
+          link: "https://github.com/open-frames/awesome-open-frames",
+        },
+      ],
+    },
+  ],
+  socials: [
+    {
+      icon: "github",
+      link: "https://github.com/open-frames/standard",
+    },
+    {
+      icon: "warpcast",
+      link: "https://warpcast.com/~/channel/open-frames",
+    },
+    {
+      icon: "x",
+      link: "https://x.com/openframesxyz",
+    },
```

Comment:
We have the Lens @openframes handle and can be reached on XMTP via openframes.eth - perhaps we can include those for now as well